src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Buttons/index.tsx

```diff
@@ -5,57 +5,41 @@ import { useField, useFormState } from 'react-final-form'
 import Button from 'src/components/layout/Button'
 import Row from 'src/components/layout/Row'
 import { styles } from 'src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/style'
-import { createTxObject } from 'src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils'
+import { isReadMethod } from 'src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils'
 
 const useStyles = makeStyles(styles as any)
 
-const Buttons = ({ onCallSubmit, onClose }) => {
+const Buttons = ({ onClose }) => {
```

Comment:
could you please add prop types for this component?

Reply:
done, also added props to most of the Contract Interaction components